lib/Rex/Commands/File.pm

```diff
@@ -357,6 +372,19 @@ sub file {
     }
 
     return \@ret;
+  } 
+  elsif ( my @exp = $file =~ m/^((?:\\.|[^\\\{])*)({(?:\\.|[^\\\*?\[])*})((?:\\.|[^\\\}])*)$/ ) {
+    if ( $exp[1] =~ /{[^},]*,[^}]*}/ ) {
+      my @ret;
+      # quote braces to keep macros
+      $exp[1] =~ s/({[^,}]*)}/\\$1\\}/g;
+  
+      for my $f ( glob $exp[1] ) {
+          push ( @ret, file( "$exp[0]$f$exp[2]", @options) );
+      }
+
+      return \@ret;
+    }
```

Comment:
No problem at all :) I'm looking forward to YAPC::EU as well ^^

I think I get it now what this branch tries to do: if the passed filename looks like a glob expression, it only wans to process the ones that has curly braces in it (because it doesn't really a good idea to process `/tmp/*` and run on a list of filenames that is there at the moment).

Maybe we can rephrase the feature's idea: support filenames expressions with curly braces in it, like if they were processed by blob.

In that case, it might be enough to detect the case "if the filename has curly braces that looks like a glob expression", and then "generate a list of files like glob would do". In this case all the filenames that is supposed to have a literal curly brace in them, needs an escaped curly brace.

I need to think about that a bit more.